src/structures/ClientApplication.js

```diff
@@ -121,9 +121,7 @@ class ClientApplication extends Base {
 
   /**
    * A link to the application's icon.
-   * @param {Object} [options={}] Options for the icon url
-   * @param {string} [options.format='webp'] One of `webp`, `png`, `jpg`
-   * @param {number} [options.size=128] One of `128`, `256`, `512`, `1024`, `2048`
+   * @param {ImageURLOptions} [options] Options for the Image URL
```

Comment:
Should be `[options={}]` since it defaults to an empty object. Same for other occurrences.

Reply:
fixed in fa58ed0